src/test/java/io/neonbee/endpoint/health/HealthEndpointTest.java

```diff
@@ -0,0 +1,80 @@
+package io.neonbee.endpoint.health;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.neonbee.NeonBee;
+import io.neonbee.NeonBeeOptions;
+import io.neonbee.health.HazelcastClusterHealthCheck;
+import io.neonbee.health.MemoryHealthCheck;
+import io.neonbee.test.helper.SystemHelper;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpClientRequest;
+import io.vertx.core.http.HttpClientResponse;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.junit5.Checkpoint;
+import io.vertx.junit5.Timeout;
+import io.vertx.junit5.VertxExtension;
+import io.vertx.junit5.VertxTestContext;
+
+@Disabled
+@ExtendWith(VertxExtension.class)
+class HealthEndpointTest {
+    private static int port;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        port = SystemHelper.getFreePort();
+    }
+
+    @Test
+    @Timeout(value = 1, timeUnit = TimeUnit.MINUTES)
+    @DisplayName("registers default and clustered checks on /health endpoint")
```

Comment:
It is intended to test the health feature end to end. After our discussion earlier I think it would be valuable to have more than one node deployed. Need to check if that is possible with the test base, though.

Reply:
We have already checked that collectHealthCheckResults works in cluster mode, so there is no need to test this in this test again.